view/execution_tree.xml

```diff
@@ -8,6 +8,7 @@ contains the full copyright notices and license terms. -->
     <field name="duration">
         <suffix name="duration" string="s"/>
     </field>
+    <field name="traceback"/>
```

Comment:
I don't think it's necessary to show the traceback in the tree, but don't have a problem either.